addons/l10n_ir_accounting/data/account.group.template.csv

```diff
@@ -0,0 +1,74 @@
+id,name,code_prefix_start,code_prefix_end,chart_template_id/id,parent_id/id
+account_group_11,دارایی های جاری,11,,l10n_ir_accounting.l10n_ir_chart_template,
```

Comment:
same here name must be in english and then translated in the po, and the id could be shorter